docs/docs/users/reference/env_variables.md

```diff
@@ -28,6 +28,7 @@ process.
 | `FOREST_MAX_FILTER_HEIGHT_RANGE`                          | positive integer                 | 2880                                           | 2880                                                          | The maximum filter height range allowed, a conservative limit of one day         |
 | `FOREST_STATE_MIGRATION_THREADS`                          | integer                          | Depends on the machine.                        | 3                                                             | The number of threads for state migration thread-pool. Advanced users only.      |
 | `FOREST_CONFIG_PATH`                                      | string                           | /$FOREST_HOME/com.ChainSafe.Forest/config.toml | `/path/to/config.toml`                                        | Forest configuration path. Alternatively supplied via `--config` cli parameter.  |
+| `FOREST_TEST_RNG_FIXED_SEED`                              | non-negative integer             | empty                                          | 0                                                             | Override RNG with a reproducible one seeded by the value                         |
```

Comment:
Let's add a note that it should be used strictly for testing to make it more obvious.

Reply:
Done.